sbi/inference/snle/snle_a.py

```diff
@@ -92,17 +66,13 @@ def __call__(
         exclude_invalid_x: bool = True,
         discard_prior_samples: bool = False,
         retrain_from_scratch_each_round: bool = False,
+        show_train_summary: bool = False,
     ) -> NeuralPosterior:
         r"""Run SNLE.
 
-        Return posterior $p(\theta|x)$ after inference.
+        Train the density estimator to learn the distribution $p(x|\theta)$.
```

Comment:
Doesn't it also return the density estimator? better add the first docstring line.